Tests/UserTests.cs

```diff
@@ -177,5 +179,77 @@ public void CanCreateUpdateAndDeleteIdentities()
             Assert.True(api.Users.DeleteUserIdentity(userId, identityId));
             Assert.True(api.Users.DeleteUser(userId));
         }
+
+        [Test]
+        public void CanMergeUsers()
+        {
+            var user1 = new User();
+            user1.Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N");
+            user1.Email = Guid.NewGuid().ToString("N") + "@" + Guid.NewGuid().ToString("N") + ".com";
+
+            var user2 = new User();
+            user2.Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N");
+            user2.Email = Guid.NewGuid().ToString("N") + "@" + Guid.NewGuid().ToString("N") + ".com";
+
+            var resultUser1 = api.Users.CreateUser(user1);
+            var resultUser2 = api.Users.CreateUser(user2);
+
+            var mergedUser = api.Users.MergeUser(resultUser1.User.Id.Value, resultUser2.User.Id.Value);
+            Thread.Sleep(2000);
```

Comment:
Please change to be a async test  and remove the thread.Sleep 

This will make it more stable 

http://simoneb.github.io/blog/2013/01/19/async-support-in-nunit/